.travis.yml

```diff
@@ -6,6 +6,7 @@ sudo: false
 cache: ccache
 
 script:
+- rm -f `which python3.6` `which python3.7`
```

Comment:
This is not robust. What if the environment is changed to add `python3.8`?

And what if a similar problem occurs in an environment that isn't Travis? As far as I understand, the goal is to work around a bug in CMake's `find_package` command. Ok, a workaround doesn't have to be pretty. But I'd expect the workaround to be in `CMakeLists.txt`, not in the Travis configuration.

Reply:
Now, I find Python 3.5 exact in travis.
Is it OK?

Reply:
No, that's no better. It's still based on “is this running on travis?” and not based on finding a python version that works.

Reply:
> BTW, cmake 3.15 has option Python3_FIND_STRATEGY=LOCATION to continue scan more python version in this case.

there is a solution based on finding a python version that works.
but it require cmake 3.15 which is not exists on travis.

Reply:
Which cmake version does one get when installing cmake with apt on travis?

```diff
 addons:
   apt:
     packages:
     - doxygen
     - graphviz
+    - cmake
```

Reply:
3.12